bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -79,6 +81,44 @@ trait UtilBijections {
   }
 
   /**
+   * Injection from twitter futures to java futures.
```

Comment:
Agreed to remove the implicit and for the doc.
I still think it's useful based on @mosesn remark that if someone needs only to go `TwitterFuture` => `JavaFuture` it makes sense to expose something which doesn't take any useless implicits.